website/blog/2024-08-12-release-0.75.md

```diff
@@ -0,0 +1,487 @@
+
+#### Moving the template to react-native-community/template
+
+Historically, `react-native` used to ship a `/template` folder inside the NPM package which was used by the Community CLI to create new projects.
+
+With our latest recommendation to use a framework, we feel that shipping an opinionated template inside our core NPM package was not aligned with [our vision](https://github.com/react-native-community/discussions-and-proposals/blob/main/proposals/0759-react-native-frameworks.md).
+
+Therefore, we decided to move the template to the [`@react-native-community/template`](https://www.npmjs.com/package/@react-native-community/template) package.
+
+This change brings the template closer to the Community CLI and will make it easier for everyone to inspect and depend on the template as a separate package.
+
+This change should be completely transparent to users who create new projects using the Community CLI. From now on, new issues related to the template should be reported on the [template issue tracker](https://github.com/react-native-community/template/issues?q=sort%3Aupdated-desc+is%3Aissue+is%3Aopen). All the various tools that depend on the template, such as the upgrade-helper, have also been updated accordingly and will continue working as usual.
+
+#### Sunsetting react-native init
+
+Similarly to the template, the `react-native init` command was also adapted to align with the new recommendation.
+
+We believe that `react-native init` is a key entry point for new developers trying React Native, and we want to offer our best recommendation to the users invoking it.
+
+So you can still use `react-native init` today to create new projects using the Community CLI & template today, but you’ll be seeing a warning as the follows:
+
+![Init Deprecation](../static/blog/assets/0.75-deprecation.png)
+
+Starting from December 31st 2024, the init command won’t create a project anymore, and you’ll have to either:
+
+- Use a framework such as Expo, with its own dedicate command to create a new project such as `npx create-expo-app`
+- Invoke the Community CLI directly with `npx @react-native-community/cli init`
+
+Please note that `react-native config` and all the other commands than `init` will continue working as usual.
+
+:::info
+In order to offer a smoother migration experience, the `react-native@0.75.0` package is still depending on `@react-native-community/cli` but we’re planning on removing this dependency in a future version of React Native.
+:::
+
+#### Auto-linking performance improvements
+
+During this work in updating the `init` command, we also spent time rewriting the auto-linking logic to be more performant. This results in faster build speed for both Android and iOS.
+
+With React Native 0.75, the auto-linking step is now running twice as fast on iOS and ~9 times faster on Android. You can read more about these improvements in this dedicated post [here](https://github.com/react-native-community/discussions-and-proposals/discussions)
```

Comment:
```suggestion
With React Native 0.75, if you are using Expo, the auto-linking step could now be running ~1.5 as fast on iOS and ~6.5 times faster on Android. You can read more about these improvements in this dedicated post [here](https://github.com/react-native-community/discussions-and-proposals/discussions/814)
```

trying to average the faster time from my benchmark.
(1.35 + 1.64) / 2 ~= 1.5
(2.88 + 10.24) / 2 ~= 6.5